docs/source/getting-started/quickstart.rst

```diff
@@ -114,7 +113,7 @@ Run the application using prototyping tools
 
 In this section, you will run the quickstart application and get introduced to the main tools for prototyping DAML:
 
-#. To compile the DAML model, run ``da run damlc -- package daml/Main.daml target/daml/iou``
+#. To compile the DAML model, run ``daml package -o target/daml/iou.dar``
```

Comment:
Wasn't there a PR lately that replaces `target` with `dist`?

Reply:
The default if you don’t specify an output path is in `dist` (and will also have a different dar name) but the quickstart example (in particular the codegen) rely on this location so for now I don’t want to change this.

Reply:
Makes sense. Thanks for the explanation.